v3/library-after-success.sh

```diff
@@ -14,16 +24,31 @@ if [[ "$TRAVIS_SECURE_ENV_VARS" == "true" ]]; then
   # Only run releases during a git tag build.
   if [[ -n "$TRAVIS_TAG" ]]; then
 
-    # Allow package.json to specify a custom build script.
-    if npm run | grep -q build:ci; then
-      echo -e "Running custom build step... `npm run build:ci`";
-      npm run build:ci
+    # Install the TravisCI CLI so we can easily make API calls.
+    gem install travis
+
+    # Read the uppercase GitHub username that initiated the build.
+    CREATED_BY=$(travis raw /v3/build/$TRAVIS_BUILD_ID --json --skip-completion-check | jq -r '.created_by.login' | tr 'a-z' 'A-Z' )
```

Comment:
What do you think about adding this logic to the `./after-success.sh` file instead? We have a few Node.js libraries calling that file directly, and not going through the `./v3/library-after-success.sh` file. 
See:
https://github.com/blackbaud/skyux-sdk-e2e/blob/master/.travis.yml#L16
https://github.com/blackbaud/auth-client/blob/master/.travis.yml#L22
https://github.com/blackbaud/help-client/blob/master/.travis.yml#L22

Reply:
Good catch.  I'm struggling to think of a way to share that logic between the two files, but nothing good is presenting itself.  I may have to repeat it.

Reply:
Strike that.  Just realized the library calls the `after-success.sh` to actually publish, so I should be able to just move the logic there!